App.js

```diff
@@ -65,6 +67,10 @@ const App = () => {
       // we hide the splash screen once we know the root view has already
       // performed layout.
       await SplashScreen.hideAsync();
+
+      // saved data in the sue module is deleted if the application returns to the foreground from
+      // the background or inactive state
+      await AsyncStorage.removeItem(SUE_REPORT_VALUES);
```

Comment:
i suggest placing this before hiding the splash screen to be sure that this process of deletion is done, before "starting" the app